README.md

```diff
@@ -1,22 +1,122 @@
 # Ethereum Alarm Clock
 
 [![Join the chat at https://gitter.im/pipermerriam/ethereum-alarm-clock](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/pipermerriam/ethereum-alarm-clock?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://travis-ci.org/chronologic/ethereum-alarm-clock.svg?branch=master)](https://travis-ci.org/chronologic/ethereum-alarm-clock)
```

Comment:
Hrm, this build status is pointing at the chronologic repo, should be updated to the new location.

Reply:
Changed